subgraphs/uniswap-v3-forks-swap/protocols/uniswap-v3-swap/config/deployments/uniswap-v3-swap-optimism/configurations.json

```diff
@@ -6,6 +6,6 @@
   "nonFungiblePositionManagerAddressStartBlock": 0,
   "deployment": "UNISWAP_V3_OPTIMISM",
   "graftEnabled": false,
```

Comment:
shouldn't grafting be enabled for these new graft changes to take effect?

Reply:
We avoid merging `graftEnabled: true` into master